src/Network/Network/ChangeLog.md

```diff
@@ -31,6 +31,7 @@
     - `New-AzPublicIpAddress`
 * Fixed bug where HubRoutingPreference didn't show up when running 'Get-AzRouteServer'
 * Updated `New-AzVirtualNetworkGateway` to remove validation for `ExtendedLocation` parameter
+* Added support for new Application Gateway SKU type, Basic SKU
```

Comment:
Please move line 34 to line 22 (that means adding your line right after "## Upcomming Release").
After it, there will be a merge conflict since ChangeLog.md has been updated during last 2 releases. Please rebase to latest main branch to merge the latest main branch. After it, the CI will succeed.